pytest_flake8.py

```diff
@@ -213,21 +213,12 @@ def check_file(path, flake8ignore, maxlength, maxdoclenght, maxcomplexity,
     if statistics:
         args += ['--statistics']
     app = application.Application()
-    prelim_opts, remaining_args = app.parse_preliminary_options(args)
-    config_finder = config.ConfigFileFinder(
-        app.program,
-        prelim_opts.append_config,
-        config_file=prelim_opts.config,
-        ignore_config_files=prelim_opts.isolated,
-    )
```

Comment:
In https://github.com/tholo/pytest-flake8/pull/88 the code is kept 🤔 but from the commit where they change they remove it.